tailwind.config.js

```diff
@@ -1,10 +1,17 @@
 module.exports = {
-  content: [
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
-  ],
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        shadow: 'linear-gradient(to bottom, rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 1) 100%)',
```

Comment:
As we already have a custom style dedicated to the background image, what about trying to setup the background image URL here (instead of using an HTML `<img` tag? :)

Reply:
@malparty , I actually opted out of background image url in order to be able to use the image optimization feature provide by `next/image`, here is the [conversation](https://github.com/vercel/next.js/discussions/18357#discussioncomment-662118) from nextjs repo!

Reply:
Nice!! That's good to me then!! :)